components/collection/drop/TokenImportButton.vue

```diff
@@ -54,7 +60,11 @@ const handleTokenImport = async () => {
 
   const api = await getApi()
   const to = urlPrefix.value === 'ahk' ? 'Statemine' : 'Statemint'
-  const call = Builder(api).to(to).amount(1e10).address(accountId.value).build()
+  const call = Builder(api)
+    .to(to)
+    .amount(price)
```

Comment:
`props.price` maybe?

```suggestion
    .amount(props.price)
```

Reply:
🕵️

Reply:
Updated. Thank you for pointing out.